pkg/minikube/cluster/cluster.go

```diff
@@ -112,6 +113,17 @@ func StartHost(api libmachine.API, config cfg.MachineConfig) (*host.Host, error)
 	s, err := h.Driver.GetState()
 	glog.Infoln("Machine state: ", s)
 	if err != nil {
+		if driver.BareMetal(config.VMDriver) || config.VMDriver == driver.VirtualBox {
```

Comment:
Lets add a comment before the if statement, stating an example when this happens ( ctrl+c...)

Reply:
Thank you for your review!
With the comment, It's easier to understand for developer as you mentioned.
I added comment about the situation.